.github/workflows/rector-cs.yml

```
@@ -25,4 +25,4 @@ jobs:
       token: ${{ secrets.YIISOFT_GITHUB_TOKEN }}
     with:
       repository: ${{ github.event.pull_request.head.repo.full_name }}
-      php: '8.4'
+      php: '8.5'
```

```suggestion
      php: '8.1'
```

PHP CS Fixer recommends use minimal version of PHP

---

Done.